### ISSUE TITLE
It seems to be an issue with python3.8

### ISSUE THREAD

**Issue by kotavi** [hdw868/pytest-aggreport#2]
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32

I get this traceback
```Traceback (most recent call last):
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest\__main__.py", line 7, in <module>
    raise SystemExit(pytest.console_main())
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\config\__init__.py", line 180, in console_main
    code = main()
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\config\__init__.py", line 157, in main
    ret = config.hook.pytest_cmdline_main(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\main.py", line 289, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\main.py", line 277, in wrap_session
    config.hook.pytest_sessionfinish(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 203, in _multicall
    gen.send(outcome)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\terminal.py", line 813, in pytest_sessionfinish
    self.config.hook.pytest_terminal_summary(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest_aggreport\plugin.py", line 195, in pytest_terminal_summary
    terminalreporter.line(self.terminal_table)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\terminal.py", line 474, in line
    self._tw.line(msg, **kw)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\_io\terminalwriter.py", line 168, in line
    self.write(s, **markup)
  File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\_io\terminalwriter.py", line 144, in write
    current_line = msg.rsplit("\n", 1)[-1]
AttributeError: 'BeautifulTable' object has no attribute 'rsplit'
```

**Comment by hdw868** [comment]
Thanks for the feedback, it seems the dependency package beautifultable changed its interface, I will take a look and fix it

**Comment by hdw868** [comment]
> Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)] on win32
> 
> I get this traceback
> 
> ```
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 193, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 86, in _run_code
>     exec(code, run_globals)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest\__main__.py", line 7, in <module>
>     raise SystemExit(pytest.console_main())
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\config\__init__.py", line 180, in console_main
>     code = main()
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\config\__init__.py", line 157, in main
>     ret = config.hook.pytest_cmdline_main(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
>     return self._hookexec(self, self.get_hookimpls(), kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
>     return self._inner_hookexec(hook, methods, kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
>     self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 208, in _multicall
>     return outcome.get_result()
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 80, in get_result
>     raise ex[1].with_traceback(ex[2])
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 187, in _multicall
>     res = hook_impl.function(*args)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\main.py", line 289, in pytest_cmdline_main
>     return wrap_session(config, _main)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\main.py", line 277, in wrap_session
>     config.hook.pytest_sessionfinish(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
>     return self._hookexec(self, self.get_hookimpls(), kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
>     return self._inner_hookexec(hook, methods, kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
>     self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 203, in _multicall
>     gen.send(outcome)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\terminal.py", line 813, in pytest_sessionfinish
>     self.config.hook.pytest_terminal_summary(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\hooks.py", line 286, in __call__
>     return self._hookexec(self, self.get_hookimpls(), kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 93, in _hookexec
>     return self._inner_hookexec(hook, methods, kwargs)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\manager.py", line 84, in <lambda>
>     self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 208, in _multicall
>     return outcome.get_result()
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 80, in get_result
>     raise ex[1].with_traceback(ex[2])
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\callers.py", line 187, in _multicall
>     res = hook_impl.function(*args)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest_aggreport\plugin.py", line 195, in pytest_terminal_summary
>     terminalreporter.line(self.terminal_table)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\terminal.py", line 474, in line
>     self._tw.line(msg, **kw)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\_io\terminalwriter.py", line 168, in line
>     self.write(s, **markup)
>   File "C:\Users\z0047w8e\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\_io\terminalwriter.py", line 144, in write
>     current_line = msg.rsplit("\n", 1)[-1]
> AttributeError: 'BeautifulTable' object has no attribute 'rsplit'
> ```

I've fixed the issue within latest pytest=6.2.2, you can take a try now.
